docker-compose.yml

```diff
@@ -1,6 +1,73 @@
 version: '3'
 
 services:
+    airflow-flower:
+        build:
+          context: ./workflow/airflow_libero
+        depends_on:
+            - airflow-redis
+        environment:
+            - EXECUTOR=Celery
+        ports:
+            - "5555:5555"
+        command: flower
+    airflow-postgres:
+        image: postgres:9.6
+        environment:
+            - POSTGRES_USER=airflow
+            - POSTGRES_PASSWORD=airflow
+            - POSTGRES_DB=airflow
+            - PGDATA=/var/lib/postgresql/data
+        volumes:
+            - 'airflow-postgres:/var/lib/postgresql/data'
+    airflow-redis:
+        image: 'redis:3.2.7'
+    airflow-scheduler:
+        build:
+          context: ./workflow/airflow_libero
+        depends_on:
+            - airflow-webserver
+        volumes:
+            - './workflow/airflow_libero/dags:/usr/local/airflow/dags'
+            - './workflow/airflow_libero/plugins:/usr/local/airflow/plugins'
+        environment:
+            - LOAD_EX=n
+            - FERNET_KEY=46BKJoQYlPPOexq0OhDZnIlNepKFf87WFwLbfzqDDho=
+            - EXECUTOR=Celery
+        command: scheduler
+    airflow-webserver:
```

Comment:
This is just the UI? Could not include it?

Reply:
Certainly, do not 'need' it, but it is very useful whilst testing/experimenting. I think the benefit outweighs the cost of the container for the skeleton.

Reply:
Could add it to a `docker-compose.override.yml`. Though I'm probably just overthinking.